tests/data/context/capability.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2022 Copyright DB Netz AG and the capellambse-context-diagrams contributors
+# SPDX-License-Identifier: Apache-2.0
+
+graph: targets * exchanges -> Capability -> exchanges * targets
```

Comment:
Like you explained to me orally, the `graph:` is currently not used anywhere. What are the plans for it going forward? Will it start to be used in the immediate future, i.e. before this PR gets merged? If not, I would be for removing it from these files, as it (seemingly) contains the same information as the block below it anyways.